app/models/fiber_channel_address.rb

```diff
@@ -0,0 +1,5 @@
+class FiberChannelAddress < Address
+  def get_address_string()
+    "fc"
```

Comment:
Don't return the string "fc". Return the value of the `fc` field.

Reply:
done